evm_arithmetization/src/cpu/decode.rs

```diff
@@ -59,6 +58,7 @@ const COMBINED_OPCODES: [usize; 11] = [
     COL_MAP.op.pc_push0,
     COL_MAP.op.m_op_32bytes,
     COL_MAP.op.push_prover_input,
+    COL_MAP.op.poseidon,
```

Comment:
I think you could leave it above as it was, and change it as
```rust
(0x22, 1, true, COL_MAP.op.poseidon),
```

to account for the flag combination